toolchain/check/import_ref.cpp

```diff
@@ -658,41 +666,22 @@ static auto SetConstantValue(Context& context, SemIR::InstId inst_id,
 
 // Adds an imported instruction without setting its constant value. The
 // instruction should later be updated by either `SetConstantValue` or
-// `ReplaceImportedInstAndSetConstantValue`.
+// `ReplacePlaceholderImportedInst`.
 template <typename InstT>
 static auto AddPlaceholderImportedInst(ImportContext& context,
                                        SemIR::InstId import_inst_id, InstT inst)
     -> SemIR::InstId {
   auto inst_id = context.local_insts().AddInNoBlock(MakeImportedLocIdAndInst(
       context.local_context(), AddImportIRInst(context, import_inst_id), inst));
   CARBON_VLOG_TO(context.local_context().vlog_stream(),
-                 "AddImportedInst: {0}\n", static_cast<SemIR::Inst>(inst));
+                 "AddPlaceholderImportedInst: {0}\n",
+                 static_cast<SemIR::Inst>(inst));
```

Comment:
```suggestion
                 "AddPlaceholderImportedInst: {0}\n", inst);
```
Drive-by, but I noticed below that you're passing typed insts directly into another formatting function, and it looks like we support that for typed instructions, so this cast seems unnecessary and inconsistent with the new code below.